pyaedt/edb_core/edb_data/components_data.py

```diff
@@ -796,6 +796,21 @@ def placement_layer(self):
         """
         return self.edbcomponent.GetPlacementLayer().Clone().GetName()
 
+    @property
+    def is_top(self):
+        """Check if a component is placed at top layer.
```

Comment:
```suggestion
        """Check if a component is placed on the top layer.
```

Reply:
thanks I changed property name and description to avoid confusion